cluster/sync.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -e
+
+cdi=$1
+cdi="${cdi##*/}"
+
+echo $cdi
+
+source ./hack/build/config.sh
+source ./cluster/gocli.sh
+
+registry_port=$($gocli ports registry | tr -d '\r')
+registry=localhost:$registry_port
+
+DOCKER_REPO=${registry} make build
```

Comment:
this should be `make docker` to generate fresh images

Reply:
done